chainerio/__init__.py

```diff
@@ -68,60 +87,194 @@ def open(file_path: str, mode: str = 'rb',
         errors, newline, closefd, opener)
 
 
-def set_root(uri_or_handler: str) -> None:
+def set_root(uri_or_handler: Union[str, Type['IO']]) -> None:
+    """Sets the current context to ``uri_or_handler``
+
+    The context here refers to the default handler and the root path.
+    The default handler points to a filesystem or a container which
+    ChainerIO uses when called without a full URI. Overrides by the full URI.
+    The default handler can be created by :func:`create_handler` with
```

Comment:
"default handler" and just a handler is a different term in this context. Does this mean an argument of `set_root` or the handler stored in the context?